resources/js/Pages/Galleries/Index.tsx

```diff
@@ -24,6 +24,9 @@ interface Galleries {
     mostValuable: App.Data.Gallery.GalleryData[];
 }
 
+const url = (filter: "most-popular" | "newest" | "most-valuable"): string =>
+    route("filtered-galleries.index", { filter });
+
```

Comment:
i don't, linter that somebody else installed was screaming at me that i was copying literal string too many times

Reply:
idk, maybe we tweak the linter then?

Reply:
sure i will just disable it because i don't find it useful, but just wanna point out that when it comes to the unnecessary complexity of our UI layer, this is the least of our worries